JS-Algorithms/challenges.js

```diff
@@ -96,6 +139,18 @@ Since 10! === 3628800 and you sum 3 + 6 + 2 + 8 + 8 + 0 + 0
 
 const digitSum = (n) => {
   // YOUR CODE HERE...
+
+  const GetFactorial = ({ n }) => {
+    return Array.from({ length: n }, (_, i) => BigInt(i + 1)).reduce(
```

Comment:
Array.from can be slower for largest numbers a simple for loop can be enough for calculating factorial